schema/swa-cli.config.schema.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "configurations": {
+      "additionalProperties": {
+        "allOf": [
+          {
+            "properties": {
+              "apiLocation": {
```

Comment:
`apiLocation` and `app` should follow the same naming convention. This would make it less confusing for users.

Either (`apiLocation` and `appLocation`) or (`api` and `app`).

Reply:
I think the issue comes from this:

`--api` and `--api-location` are the same option. Check my above comment for why I'd like to use `apiLocation`.

However, `--app` and `--app-location` are NOT the same option. As one might expect. Rather `--app` and `--app-artifact-location` are the same. And `--app-location` is a separate option from `--app`.

So we actually need three:
1. `app` or `appArtifactLocation`
2. `api` or `apiLocation`
3. `appLocation`

Reply:
See https://github.com/Azure/static-web-apps-cli/pull/294#discussion_r711582369